kostal/__init__.py

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env python3
 # vim: set encoding=utf-8 tabstop=4 softtabstop=4 shiftwidth=4 expandtab
-#
-# Copyright 2013 KNX-User-Forum e.V.            http://knx-user-forum.de/
-#
-#  This file is part of SmartHomeNG.    https://github.com/smarthomeNG//
+#########################################################################
+#  Copyright 2017 Wenger Florian                       <wenger@unifox.at>
```

Comment:
Can you pleas add an additional copyright line:
`#  Copyright 2012-     Oliver Hinckel                  github@ollisnet.de`